bash/symfony_aliases

```diff
@@ -27,5 +27,21 @@ alias sfaw='php app/console assetic:watch'
 # Symfony assets alias
 alias sfai='php app/console assets:install web --symlink'
 
-# Symfony server alias
-alias sfs='php app/console server:run'
+# Symfony server aliases
+alias sfsr='php app/console server:run'
+alias sfss='php app/console server:status'
+alias sfsq='php app/console server:stop'
+
+# Doctrine aliases
+alias dodbcreate='doctrine:database:create'
```

Comment:
@calina-c: Shouldn't all the below alias's commands start with `php app/console`?

Reply:
To keep everything consistent, wouldn't it be better to have all Symfony related aliases start with `sf`?